### PR TITLE
Add new infos to the product query

### PR DESCRIPTION
**Pull request by ullayne02**
#### What problem is this solving?
Add regionId and salesChannel to product query

#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: graphql/schema.graphql [hunk 1]
```diff
@@ -16,6 +16,14 @@ type Query {
     Product identifier
     """
     identifier: ProductUniqueIdentifier
+    """
+    Seller id encoded with base64 according to this format SW#{selerId}
```
Comment (frateligu) [inline graphql/schema.graphql:20]: seller id ou region id? 
E Faltou um "L" no #{selerId}

Comment (ullayne02) [reply graphql/schema.graphql:20]: sellerId, regionid eh eh o seller Id no formato "SW#{sellerId}" codificado em base64

Comment (thalytafabrine) [reply graphql/schema.graphql:20]: falta corrigir o `SW#{sellerId}`

File: graphql/schema.graphql [hunk 2]
```diff
@@ -343,9 +351,13 @@ type Query {
...
     hideUnavailableItems: Boolean
     """
-    Seller id encoded with base64 according to this format SW#{selerId}
+    Seller id encoded with base64 according to this format SW#{sellerId}
     """
     regionId: String
+    """
+    Trade Policy related to the region ID
+    """
+    tradePolicy: String
```
Comment (thalytafabrine) [inline graphql/schema.graphql:360]:
```suggestion
    salesChannel: String
```

**Review by thalytafabrine — APPROVED** [review]

**Pull request merged**